src/App.tsx

```diff
@@ -1,7 +1,34 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+interface Clock {
+  pressedKey: string | null;
+}
+
+export class App extends Component<{}, Clock> {
+  state = {
+    pressedKey: null,
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', (event: KeyboardEvent) => {
+      this.setState({ pressedKey: event.key });
```

Comment:
Create separate function for `event`.
```suggestion
    document.addEventListener('keyup', this.findKey);
```